src/tuplet.ts

```diff
@@ -235,19 +195,23 @@ export class Tuplet extends Element {
   }
 
   resolveGlyphs(): void {
-    this.numeratorGlyphs = [];
+    let numerator = '';
+    let denominator = '';
     let n = this.numNotes;
     while (n >= 1) {
-      this.numeratorGlyphs.unshift(new Glyph('timeSig' + (n % 10), this.point));
-      n = parseInt((n / 10).toString(), 10);
+      numerator = String.fromCharCode(0xe880 /* tuplet0 */ + (n % 10)) + numerator;
+      n = Math.floor(n / 10);
     }
-
-    this.denominatorGlyphs = [];
-    n = this.notesOccupied;
-    while (n >= 1) {
-      this.denominatorGlyphs.unshift(new Glyph('timeSig' + (n % 10), this.point));
-      n = parseInt((n / 10).toString(), 10);
+    if (this.ratioed) {
+      n = this.notesOccupied;
+      while (n >= 1) {
+        denominator = String.fromCharCode(0xe880 /* tuplet0 */ + (n % 10)) + denominator;
+        n = Math.floor(n / 10);
+      }
+      denominator = '\uE88A' /* tupletColon */ + denominator;
```

Comment:
Can you include a couple of images in this PR discussion to illustrate how the numerator + denominator looks?

Reply:
done